gulp/ui-test.js

```diff
@@ -21,7 +21,7 @@ const launchHeadLess = function (done, files, config) {
             args: [
                 "--params.login.email", args[0],
                 "--params.login.pwd", args[1],
-                "--baseUrl", args[2]
+                "--params.baseUrl", args[2]
```

Comment:
try to just remove the initial --params and see if it deserializes correctly again